doc/api.rst

```diff
@@ -2,12 +2,40 @@
 API
 ###
 
+This is the reference for classes (``CamelCase`` names) and functions
+(``underscore_case`` names) of MNE-Connectivity, grouped thematically by analysis
+stage. MNE-Connectivity also provides connectivity data structure
+containers for different types of connectivity data, which are
```

Comment:
You sometimes call the data structures classes (below) and sometimes containers, that could potentially be confusing for novice users. Listing them extra also makes it sound to me like they are different from the classes you list first.

Maybe something like `The data structure classes contain different types of connectivity data and are described below.`?

@larsoner is there some standard way of doing this?  I see MNE does not mentions "data structure" at all and just lists the classes.